### ISSUE TITLE
Current Code Errors

### ISSUE THREAD

**Issue by enesdmc0** [adrianhajdin/portfolio#8]
Project codes are different from the current code in live, can you share the final version of the project with us?

**Comment by QuintenStr** [comment]
Same issue here... Please update! :)

**Comment by NAVI1322** [comment]
same here, Having errors with dark mode

**Comment by anjalisah89** [comment]
> same here, Having errors with dark mode

If you're having issue with theme clear the cache for localhost and try again. You can also try with other browsers to see if its a cache problem.

**Comment by kubakakauko** [comment]
> same here, Having errors with dark mode

Can you tell us what errors are you getting?

**Comment by Ghanbariii** [comment]
I have a problem with BentoGrid, my gird section has a different style than yours. I try many ways but it is not working. Please help me

**Comment by sam-arth07** [comment]
> I have a problem with BentoGrid, my gird section has a different style than yours. I try many ways but it is not working. Please help me

Same here. For some reason even after applying the same styles, I am not getting the required configuration of the bentogrid

**Comment by Imadh-Ifham** [comment]
I am having the same issue with BentoGrid, lg:col-span-3 cover the full width in lg screen. I tried nesting it into a div having a className with grid grid-cols-* but still i didn't match the requirements

**Comment by Ghanbariii** [comment]
> @Ghanbariii @sam-arth07 have you followed the tutorial or have you cloned this repository? The code here is slightly different than in the tutorial and if you followed part of the tutorial and then pulled code from here it may not be working correctly.

No I followed videos, This is really odd. after that I checked repository

**Comment by sam-arth07** [comment]
> @Ghanbariii @sam-arth07 have you followed the tutorial or have you cloned this repository? The code here is slightly different than in the tutorial and if you followed part of the tutorial and then pulled code from here it may not be working correctly.

Yes i did follow the tutorial but maybe i missed out on the updated styling that is here on the github repository. I think it is not all about code along. Sometimes you have to take a step back and try to solve the problems by yourself. By the way, thank you for pointing out the anomaly.
